1. Desktop app calls Web API/TodoListService/Controllers/TodoListController.cs

```diff
@@ -41,6 +42,7 @@ public class TodoListController : Controller
         [HttpGet]
         public IEnumerable<TodoItem> Get()
         {
+            HttpContext.VerifyUserHasRequiredScope("user_impersonation");
```

Comment:
>user_impersonation [](start = 52, length = 18)

do you have a const file for things like this?

Reply:
Done. Thanks!